packages/server/lib/errors.js

```diff
@@ -694,6 +694,19 @@ const getMsgByType = function (type, arg1 = {}, arg2, arg3) {
         ${chalk.yellow(arg1.error)}
 
         Learn more at https://on.cypress.io/reporters`
+      // TODO: update with vetted cypress language
+    case 'NO_DEFAULT_CONFIG_FILE_FOUND':
+      return stripIndent`\
+        Could not find a Cypress configuration file, exiting.
+
+        We looked but did not find a default config file in this folder: ${chalk.blue(arg1)}`
+      // TODO: update with vetted cypress language
+    case 'CONFIG_FILES_LANGUAGE_CONFLICT':
+      return stripIndent`
+          There is both a \`${arg2}\` and a \`${arg3}\` at the location below:
+          ${arg1}
+          Cypress does not know which one to read for config. Please remove one of the two and try again.
```

Comment:
Can we get this to show up on separate lines for better legibility?

```suggestion
          ${arg1}
          
          Cypress does not know which one to read for config. Please remove one of the two and try again.
```
<img width="767" alt="Screen Shot 2021-09-24 at 11 00 59 AM" src="https://user-images.githubusercontent.com/1271364/134705823-593f4a4d-6095-4928-8e4d-7ed6537b06cf.png">